x-pack/plugins/cloud_security_posture/public/common/hooks/use_theme.tsx

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useTheme as useEmotionTheme } from '@emotion/react';
+import { EuiTheme } from '../../../../../../src/plugins/kibana_react/common';
+
+export function useTheme() {
```

Comment:
Lets drop this and use `useEuiTheme` from `eui` instead

Reply:
seems like `useEuiTheme` doesn't include the value of the emotion theme for `euiPageBackgroundColor`, and no similar value either.

Reply:
It exists, look at shades in the [colors documentation](https://elastic.github.io/eui/#/theming/colors)

```
colors.emptyShade

Used as the background color of primary page content and panels including modals and flyouts.
```